Client/Frontend/Settings/AppSettingsTableViewController.swift

```diff
@@ -65,6 +65,14 @@ class AppSettingsTableViewController: SettingsTableViewController {
             ]
         }
 
+        generalSettings += [
+            BoolSetting(prefs: prefs,
+                prefKey: "goToCopiedURL",
```

Comment:
Looking at the other setting boolean toggles we have, it looks like we are namespacing the settings under settings. For example, the `Close Private Tabs` option prefKey is `settings.goToCopiedURL`. I see that the other general settings are not namespaced but we should consider moving those under settings later on.